cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,31 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  const randomNumber = Math.random().toString().slice(2, 5);
```

Comment:
Generating a random number for each test run can make the test results inconsistent. Consider using a static test data or use a setup function to create necessary data before each test run.